woocommerce/wc-admin
# Bump Version to 2.2.0-dev

Bumps versions to `2.2.0-dev` now that the 2.1 branch has been cut.

No changelog is needed

### Testing Instructions

- Navigate to WooCommerce -> Status
- Ensure a value of `2.2.0-dev` for WooCommerce Admin package. 

<img width="508" alt="Screen Shot 2021-02-26 at 7 26 59 am" src="https://user-images.githubusercontent.com/9312929/109233039-258afc80-7804-11eb-8239-7151e73364bf.png">
